linera-indexer/src/indexer.rs

```diff
@@ -125,10 +125,10 @@ where
         let mut value = value.clone();
         loop {
             let Some(block) = value.inner().block() else {
-                break;
+                return Err(IndexerError::InvalidBlock(value.hash()));
```

Comment:
Let's call it `InvalidCertificateValue` or `InvalidValue`. Because in this case it's precisely _not_ a block.